sound/soc/sof/sof-client-dma-trace.c

```diff
@@ -291,6 +291,32 @@ static size_t sof_wait_trace_avail(struct sof_client_dev *cdev,
 	return sof_trace_avail(cdev, pos, buffer_size);
 }
 
+static int sof_dfsentry_trace_open(struct inode *inode, struct file *file)
+{
+	struct sof_client_dev *cdev;
+	int ret;
+
+	ret = debugfs_file_get(file->f_path.dentry);
+	if (unlikely(ret))
+		return ret;
+
+	cdev = inode->i_private;
+
+	ret = sof_client_core_module_get(cdev);
```

Comment:
I don't get why this is required.

Reply:
The `debugfs_file_get()` is enough, I'll remove the `module_get()` as it looks like that things got sorted out along the way even if the parent is removed.